recipe/meta.yaml

```diff
@@ -42,6 +43,8 @@ requirements:
   run:
     - r-base
     - {{native}}gcc-libs         # [win]
+    - libcxx                     # [osx and c_compiler == "clang"]
```

Comment:
this line is my only concern, I do not get why this is needed.
But yes please feel free to merge.

Reply:
Without libcxx, there is the waning:
```
WARNING (r-openimager,lib/R/library/OpenImageR/libs/OpenImageR.so): Needed DSO lib/libc++.1.dylib found in ['libcxx']
WARNING (r-openimager,lib/R/library/OpenImageR/libs/OpenImageR.so): .. but ['libcxx'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

See https://travis-ci.org/conda-forge/r-openimager-feedstock/jobs/474559815#L639

This was observed while working on r-hypergea and r-lightgbm.

I added now two commits here the according commit message to remove and re-add libcxx again so that it is in the git history here.

Reply:
Or is that a false positive, that is not required as runtime requirement despite the warning?

Reply:
Have the test passed before? I guess this is overlinking and can be ignored?

Reply:
Yes, they passed... will remove the re-add commit. Thanks.